xlint

```diff
@@ -19,15 +19,25 @@ header() {
 	fi
 }
 
+exists_once_for() {
+	local var=$1
+	case "$(grep -c "^${var}=" "$template")" in
+		0) echo "$template: '$var' missing!";;
+		1) ;;
+		*) echo "$template: '$var' defined more than once";;
+	esac
+}
+
 exists_once() {
 	for var in pkgname version revision short_desc maintainer license \
 		   homepage; do
-		case "$(grep -c "^${var}=" "$template")" in
-			0) echo "$template: '$var' missing!";;
-			1) ;;
-			*) echo "$template: '$var' defined more than once";;
-		esac
+		exists_once_for "$var"
 	done
+	if grep -Eq "^build_style=\(['\"]\?\)meta\1" "$template"; then
+		for var in distfiles checksum SOURCE_DATE_EPOCH; do
```

Comment:
This is good change, independently of SOURCE_DATE_EPOCH. But it should be rather `exists_at_most_once_for` because more build styles do not require these vars, like `R-cran`.